interoperability/algohosting.md

```diff
@@ -57,6 +57,8 @@ In general, the aim is to simplify the process of building high-quality on-deman
 +-------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 | PROV-REQ-17 | Executables shall offer at least one choice of a non-interactive command line interface, or an API for integration into a larger codebase.                                                                |                                                                                                                                                                                                                                                                                                              |
 +-------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| PROV-REQ-18 | Algorithms shall be associated with and tested on at least one APEx compliant hosting platform.                                                                |                                                                                                                                                                                                                                                                                                              |
++-------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
```

Comment:
```suggestion
+-------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| PROV-REQ-18 | Algorithms shall be associated with and tested on at least one APEx compliant hosting platform.                                                                                                           |                                                                                                                                                                                                                                                                                                              |
|             | An overview of APEx-compatible platforms is available [here](../propagation.md#what-are-the-platforms-to-choose-from).                                                                                    |                                                                                                                                                                                                                                                                                                              |
+-------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+

```

Reply:
I've added a reference to the list of APEx-compatible platforms

Reply:
makes sense, but be careful: this is a 'normative' specification, which means that it should be regarded as the source of truth.
If that list is also normative, (meaning guaranteed to be up-to-date), then it's fine, but the thing is that we simply cannot know the 'full' list of compliant platforms, unless if we would be issuing compliance certificates.
But long story short, my recommendation is to specify that the list may be incomplete or should not be considered normative.

Reply:
True, the list may not be normative as there could be platforms that are compliant but we haven't used them before and therefore are not in the list.

Reply:
I have updated the comment to include a disclaimer. If it doesn't make sense, we can also ignore the suggestion. An alternative approach could be to refer to the algorithm hosting platform guidelines. I am just thinking from the perspective of the user/algorithm dev if it would make sense to state more clearly what an APEx-compliant hosting platform encompasses.